readme.md

```diff
@@ -68,33 +71,71 @@
+- [Bonsai Trees](https://www.curseforge.com/minecraft/mc-mods/bonsai-trees) - Adds essentially mini tree farms which are one block big.
+- [Ex Nihilo: Creatio](https://www.curseforge.com/minecraft/mc-mods/ex-nihilo-creatio) - A great mod for skyblocks, it filters things like sand and gravel for useful resources like iron and diamond, compatible with lots of mods. (Ex Nihilo: Creatio in latin roughly translates to Out of Nothing: I Create)
 
 ### Energy
 
 - [RFTools](https://minecraft.curseforge.com/projects/rftools) - RFTools is a mod containing various tools that are related to RF (Redflux) or technology in general.
-- [Applied Energistics 2](https://github.com/AppliedEnergistics/Applied-Energistics-2) - A Minecraft Mod about Matter, Energy and using them to conquer the world.
+- [Extreme Reactors](https://www.curseforge.com/minecraft/mc-mods/extreme-reactors) - Adds nuclear reactors with the fictional fuel 'yellorium'. A carry on of the original Big Reactors.
+- [Zetta Industries](https://www.curseforge.com/minecraft/mc-mods/zetta-industries) - A mod which adds the main multiblock of a big battery.
+
+### Magic
+
+- [AbyssalCraft](https://www.curseforge.com/minecraft/mc-mods/abyssalcraft) - AbyssalCraft is a magic mod centered around exploration.
+- [Astral Sorcery](https://www.curseforge.com/minecraft/mc-mods/astral-sorcery) - This is a magic mod based on the stars, and harnessing the powers of starlight and the constellations.
+- [Avaritia](https://www.curseforge.com/minecraft/mc-mods/avaritia) - Avaritia is a mod with a really long progression tree, but really overpowered in the late game. Its soul purpose is to drag the endgame gameplay to extreme lengths.
+- [Blood Magic](https://www.curseforge.com/minecraft/mc-mods/blood-magic) - This is a magic mod based on blood. Beware, you die a lot.
 
-### Tweaking
+### Utilities
 
 - [TNTUtils](https://github.com/ljfa-ag/TNTUtils) - More control over explosions in Minecraft.
+- [Akashic Tome](https://www.curseforge.com/minecraft/mc-mods/akashic-tome) - A mod based on Morph-o-Tool, it mainly gives the ability to merge different mods' documentation books.
+- [AppleSkin](https://www.curseforge.com/minecraft/mc-mods/appleskin) - This mod does not change any existing mechanics, but simply makes the existing hunger saturation and exhaustion levels visible in the default GUI.
+- [Attribute Fixer](https://www.curseforge.com/minecraft/mc-mods/attributefix) - This mod fixes the attribute system in Minecraft which handles things like armour points, because many other mods are inadvertantly affected by a limit in this system.
+- [Diet Hopper](https://www.curseforge.com/minecraft/mc-mods/diet-hoppers) - "This mod modifies the way Minecraft calculates the collision boxes of the hopper, so that you can access any visible blocks behind it."
```

Comment:
```suggestion
- [Diet Hopper](https://www.curseforge.com/minecraft/mc-mods/diet-hoppers) - This mod modifies the way Minecraft calculates the collision boxes of the hopper, so that you can access any visible blocks behind it.
```